### PR TITLE
Spatial index

### PR DESCRIPTION
**Pull request by mgeplf**

### CODE REVIEW

File: include/bbp/sonata/config.h [hunk 1]
```diff
@@ -183,12 +200,6 @@ class SONATA_API CircuitConfig
         std::string biophysicalNeuronModelsDir;
     };
 
-    struct SubnetworkFiles {
```
Comment (edasubert) [inline include/bbp/sonata/config.h:186]: why is this removed?

**Review by edasubert — APPROVED** [review]
seems ok to me, but it is long time since I worked with c++

File: include/bbp/sonata/config.h (hunk 1 above)
Comment (mgeplf) [reply include/bbp/sonata/config.h:186]: good eye; it wasn't used, and should have been removed before.

**Comment by mgeplf** [comment]
> seems ok to me, but it is long time since I worked with c++

Same here :)

I just wanted to make sure someone else looked at the names, to make sure it was what we decided.

**Review by matz-e — APPROVED** [review]
